homeassistant/components/igloohome/config_flow.py

```diff
@@ -0,0 +1,60 @@
+"""Config flow for igloohome integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from aiohttp import ClientError
+from igloohome_api import Auth as IgloohomeAuth, AuthException
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_CLIENT_ID, CONF_CLIENT_SECRET
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_CLIENT_ID): str,
+        vol.Required(CONF_CLIENT_SECRET): str,
+    }
+)
+
+
+class MyConfigFlow(ConfigFlow, domain=DOMAIN):
```

Comment:
```suggestion
class IglooHomeConfigFlow(ConfigFlow, domain=DOMAIN):
```

Reply:
You didn’t solve that one

Reply:
Apologies, forgot to push the changes. :3